init/ocp4.inc.sh

```diff
@@ -110,11 +110,19 @@ function ocp4_step_process_hostname
     fi
 }
 
+function ocp4_step_enable_httpd_service
+{
+    if container_helper_exist_ca_cert; then
+        systemctl is-enabled httpd || systemctl enable httpd
```

Comment:
Do we want to start it also, at this moment?  If so, `systemctl enable --now httpd` will do it.